build.gradle.kts

```diff
@@ -230,20 +234,21 @@ subprojects {
 		dependencies {
 			implementation(project(":api"))
 		}
+		unimined.minecraft(sourceSets["main"], lateApply = true) {
+			if (implName != "primitive") {
```

Comment:
Might be cleaner to check the `side` value than special-casing `primitive`

Reply:
I agree, but that's unfortunately not possible due to this being called before `side` is set (I think? not too sure but either way it doesn't work)

Reply:
If you've tried it and it doesn't work, then feel free to resolve this comment. Otherwise, please try it first.